### PR TITLE
Create meeting ical for AWS Working Group

### PR DESCRIPTION
**Pull request by alinabuzachis**
Create meeting ical for AWS Working Group -  not sure if this is enough, I'll need some guidance otherwise.

### CODE REVIEW

**Comment by alinabuzachis** [comment]
cc @tremble @markuman

File: meetings/aws.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+project: AWS working group
+meeting_id: AWS
+project_url: https://github.com/ansible/community/wiki/AWS
+agenda_url: https://github.com/ansible/community/issues?q=is:open+label:meeting_agenda+label:aws
+chair: Alina Buzachis (abuzachis), Mark Chappel (tremble), Markus Bergholz (markuman)
```
Comment (tremble) [inline meetings/aws.yaml:5]:
```suggestion
chair: Alina Buzachis (abuzachis), Mark Chappell (tremble), Markus Bergholz (markuman)
```

**Review by tremble — CHANGES_REQUESTED** [review]

File: meetings/ical/aws.ics [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+PRODID:-//yaml2ical agendas//EN
+BEGIN:VEVENT
+SUMMARY:AWS working group
+DTSTART;VALUE=DATE-TIME:20220428T173000Z
+DURATION:PT1H
+DTSTAMP;VALUE=DATE-TIME:20220316T174901Z
+UID:awsworkinggroup-20220428
+RRULE:FREQ=MONTHLY;BYDAY=4TH
+DESCRIPTION:Project:  AWS working group\nChair:  Alina Buzachis (abuzachis
+ )\, Mark Chappel (tremble)\, Markus Bergholz (markuman)\nDescription:  \nW
```
[the remark this answers is not in the document]
Comment (alinabuzachis) [reply meetings/ical/aws.ics]: Sorry for the type. Solved it.

**Review by markuman — APPROVED** [review]

**Comment by alinabuzachis** [comment]
@cybette Can we merge this if it's sensible to you? Thanks.

File: meetings/aws.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  We discuss everything related to AWS every 4th Thursday of the month at 17:30 UTC.
+  
+  If you have something to discuss (e.g. a PR that needs help),
+  add your request to the meeting agenda and join the IRC channel.
+schedule:
+- time: '1730'
+  start_date: 20220324
+  day: Thursday
+  irc: ansible-aws or or https://matrix.to/#/#aws:ansible.im
```
Comment (cybette) [inline meetings/aws.yaml:16]: one too many `or`s :)

**Pull request merged**